app/models/main.py

```diff
@@ -125,7 +125,12 @@ class Framework(db.Model):
         backref='frameworks'
     )
     allow_declaration_reuse = db.Column(db.Boolean, nullable=False, default=False)
+
     application_close_date = db.Column(db.DateTime, nullable=True)
+    intention_to_award_date = db.Column(db.DateTime, nullable=True)
+    clarifications_close_date = db.Column(db.DateTime, nullable=True)
+    clarifications_publish_date = db.Column(db.DateTime, nullable=True)
+    framework_live_date = db.Column(db.DateTime, nullable=True)
```

Comment:
If these really will be UTC then an `_utc` postfix on the column name is a good idea, as a reminder that something needs to happen before it's displayed to users.

Reply:
(And probably use `_time` rather than `_date`.)

Reply:
I've opted for `_at` rather than `_time`, as the latter feels like it could equally not carry date information - and `_datetime` feels a bit excessive.